### PR TITLE
#7 GitHub actions

### PR DESCRIPTION
**Pull request by RidwanHamid501**
Set up GitHub workflow to show pylint and pytest scores. Addresses issue #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+![Pylint Score](.github/badges/pylint.svg) 
+![Passing Tests](.github/badges/test.svg)
+
 # Trend Getter
```
Comment (SurinaCS) [inline README.md:4]: great addition to the readme

File: .github/workflows/development.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  contents: write
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+  workflow_dispatch:
```
Comment (SurinaCS) [inline .github/workflows/development.yml:13]: very clear code

**Review by SurinaCS — COMMENTED** [review]
great work, does this include the pylint and pytests all in one?

**Review by SurinaCS — APPROVED** [review]

File: .github/badges/test.svg
Comment (Kurt812) [inline .github/badges/test.svg:1]: Showing 0/0 as we have no code ready for testing yet.

**Pull request merged**